rootfs/etc/nginx/lua/util/dns.lua

```diff
@@ -147,7 +147,7 @@ function _M.lookup(host)
   end
 
   if #dns_errors > 0 then
-    ngx_log(ngx_ERR, "failed to query the DNS server:\n" .. table_concat(dns_errors, "\n"))
+    ngx_log(ngx_ERR, "failed to query the DNS server for %s:\n%s", host, table_concat(dns_errors, "\n"))
```

Comment:
you can not do string formatting like this using `ngx.log` - it does not work

Reply:
you can do following instead

```
ngx_log(ngx_ERR, "failed to query the DNS server for ", host, ":\n", table_concat(dns_errors, "\n"))
```

Reply:
thanks, resolved :)